pkgs/development/libraries/gstreamer/rs/default.nix

```diff
@@ -242,6 +242,14 @@ stdenv.mkDerivation rec {
     runHook postCheck
   '';
 
+  doInstallCheck = (lib.elem "webp" selectedPlugins) && !stdenv.hostPlatform.isStatic &&
+    stdenv.hostPlatform.parsed.kernel.execFormat == lib.systems.parse.execFormats.elf;
```

Comment:
the `!stdenv.hostPlatform.isStatic` conditional should stay, because this explicitly checks if libwebp is dynamically linked, which we do not expect in the isStatic case.

Reply:
Ah right, that makes sense. It's late and brain not working right. Ignore this review comment then 😅